running/bodies_and_contacts.md

```diff
@@ -141,4 +141,48 @@ This does not remove the contact from the FixMyStreet database (because doing
 so might break any existing problem reports that used it), but it prevents the
 contact from being used for new reports.
 
+## Deleting bodies
+
+It's unusual to need to delete a body, but it sometimes happens &mdash; for example, if a council ceases to exist because it has merged with an existing one.
+If this happens, remember that you may also need to create a new body, or
+change the 
+<a href="/glossary/#area" class="glossary__link">admin boundary</a>
+of an existing one.
+
+To delete a body, go to the body's admin page to edit it. Tick the checkbox
+marked **Flag as deleted**, and then click **Update body**.
+
+This does not remove the body from the FixMyStreet database (because doing
+so would remove all existing problem reports that used it). The reports,
```

Comment:
Not actually true, but the code certainly assumes a body associated with a report exists :)